plugins/lib/primitives-v8.json

```diff
@@ -811,6 +811,10 @@
     {
       "props": ["color", "fill"],
       "replacement": "--fgColor-default"
+    },
+    {
+      "props": ["background"],
```

Comment:
Why are we using `"props": ["background"],` with `"replacement": "--fgColor-default"`?

I think I probably don't understand how those files work.

Reply:
I think I added a few odd cases because I came across a lot of instances in dotcom where `color-fg-default` is used as a background color. My thinking here is that we make the initial swap easier to get the test going, and later we can introduce proper linting that only allows this token on `color` in most cases.

Sometimes you need a fg color for background-color where it kind of acts like a fg color though. So there will always be one off lint disables.